cargo-crev/src/edit.rs

```diff
@@ -161,9 +161,9 @@ pub fn build_trust_proof_interactively(
     local: &Local,
     from_id: &PublicId,
     ids: Vec<Id>,
-    trust_or_distrust: TrustProofType,
+    proof_type: TrustProofType,
```

Comment:
This should be just `trust_or_distrust : TrustOfDistrust` where `enum TrustOrDistrust { Trust, Distrust`. :)

Reply:
Oh. Nvmd. This is a function for creating trust proof. :D